app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -42,7 +42,7 @@ def create_usps_enrollment(enrollment, pii)
         address = [pii['address1'], pii['address2']].select(&:present?).join(' ')
         applicant = UspsInPersonProofing::Applicant.new(
           {
-            unique_id: enrollment.usps_unique_id,
+            unique_id: enrollment.unique_id || enrollment.usps_unique_id,
```

Comment:
I think I'm missing something here.  

I understand why the helper checks if `enrollment.unique_id` exists, but I don't understand why, if the unique id doesn't exist, this code still uses the deprecated `InPersonEnrollment` instance method `usps_unique_id`.  
Why isn't this written like:
```ruby
unique_id: enrollment.unique_id || enrollment.set_unique_id
```
(I realize that the above would require the `set_unique_id` method to be made public.)


Are we using the deprecated method in order to maintain backwards compatibility?

Reply:
Yes we're using `usps_unique_id` for backwards compatibility. If an enrollment doesn't have a `unique_id` then that enrollment was created in the USPS API using the value of `usps_unique_id`, so we need to use that same value here

Reply:
It's worth a comment; I'll add one

Reply:
Added a comment in 5a8f17baf

Reply:
Added a comment in 5a8f17baf